Gemfile.lock

```diff
@@ -398,23 +398,23 @@ GEM
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
-    sshkit (1.20.0)
+    sshkit (1.21.0)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tzinfo (1.2.5)
+    tzinfo (1.2.6)
       thread_safe (~> 0.1)
     uglifier (2.7.2)
       execjs (>= 0.3.0)
       json (>= 1.8.0)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.6)
-    unicode-display_width (1.6.0)
+    unicode-display_width (1.6.1)
```

Comment:
### [unicode-display_width](https://github.com/janlelis/unicode-display_width)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.6.0 → 1.6.1

[_[change-log](https://github.com/janlelis/unicode-display_width/blob/master/CHANGELOG.md), [source-code](https://github.com/janlelis/unicode-display_width)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/janlelis/unicode-display_width/compare/v1.6.0...v1.6.1).

These are the individual commits:
- (2f68760) [Update travis Rubies](https://github.com/janlelis/unicode-display_width/commit/2f687608205c78b274f1a7aeced08cc84c6665f2)
- (c918ea7) [Add Ruby 2.7 to list of supported Rubies](https://github.com/janlelis/unicode-display_width/commit/c918ea75bc9357cb2e003cfe8a213fda2988b474)
- (865845a) [travis: Allow JRuby to fail](https://github.com/janlelis/unicode-display_width/commit/865845aeed558e3303063a1e9ea0afa52914e48c)
- (7498446) [Update gemspec](https://github.com/janlelis/unicode-display_width/commit/74984467534599bb97db9ead0f3bd2a1c9ef7923)
- (5ae49c9) [Update license year](https://github.com/janlelis/unicode-display_width/commit/5ae49c9c6b80f7da77475146bd971e31848ca2ac)
- (3a73f61) [Fix that ambiguous and overwrite options where ignored for e…](https://github.com/janlelis/unicode-display_width/commit/3a73f615932aaa12854ade5d8a5c35a801822838)
- (4d3d214) [Release v1.6.1](https://github.com/janlelis/unicode-display_width/commit/4d3d214fc240428541bb76950a7519602f3a961a)


</details>